src/ESPEasyEth.cpp

```diff
@@ -98,6 +102,14 @@ void ethPrintSettings() {
   addLog(LOG_LEVEL_INFO, settingsDebugLog);
 }
 
+String ETHMacAddress() {
+    uint8_t mac[6];
+    char macStr[18] = { 0 };
+    esp_eth_get_mac(mac);
+    sprintf(macStr, "%02X:%02X:%02X:%02X:%02X:%02X", mac[0], mac[1], mac[2], mac[3], mac[4], mac[5]);
```

Comment:
There is already a `formatMAC` function in the code.

Reply:
Oh yes you are right. I will change it.

Reply:
This String function is not needed, because there is a 
`ETH.macAddress()`
which returns a String.
I changed it to the byte version of the function and now 
`uint8_t * NetworkMacAddressAsBytes(uint8_t* mac)`
returns either this in Ethernet mode or
`WiFi.macAddress(mac)`
in Wifi mode.